### DISCUSSION TITLE
Is the documentation site changed?

### DISCUSSION THREAD

**Discussion by ColdSnaap** [materialsproject/pymatgen#3216] [Q&A] [answered]
Is the documentation site changed? The new site is so hard to use compared to just a few days ago...

**Comment by shyuep** [comment]
I think it would be helpful to say what is difficult about it? The content is exactly the same.

**Reply by ColdSnaap** [reply]
Sorry my bad. Generally, I feel like it's way harder to navigate. We had a list of all the modules on the left, for someone who is not super experienced with pymatgen like me, I could get an overview of what modules might have the function I need with my work. And I could just click in all the classes would be on the main page. Even if I click the wrong module I can just go check another one. Right now everything is on the same page and there is no drop down menu. If I don't have a clear idea of what class name I need. I have to keep scrolling down on the same page... And there is no link to the source code either?
This has been my main issue after a whole day trying to navigate through the document... Thanks for answering.

**Reply by shyuep** [reply]
I have updated the docs page to include a Table of contents for each page that will help with class navigation. The API doc also includes all the modules.

For future reference, it is more helpful to precisely state your issues and even better, make suggestions on ways to improve either code or presentation. Posting something generic like "The new site is hard to use" is non-actionable and impossible for us to do anything to help.

**Reply by ColdSnaap** [reply]
Sorry this is my first time post. I thought it was casual discussion like Reddit.

**Reply by shyuep** [reply, marked as answer]
No problem. Anyway, I updated the docs. The old functionality should now be back.
